websites/M/Memrise/presence.ts

```diff
@@ -0,0 +1,254 @@
+				state: "All categories",
+				smallImageKey: Assets.Searching,
+			};
+		case "top":
+			return {
+				details: "Browsing Community forums...",
+				state: "Top posted",
+				smallImageKey: Assets.Searching,
+			};
+		case "latest":
+			return {
+				details: "Browsing Community forums...",
+				state: "Latest posts",
+				smallImageKey: Assets.Searching,
+			};
+		case "badges":
+			return {
+				details: "Browsing Community forums...",
+				state: "All badges",
+				smallImageKey: Assets.Searching,
+			};
+		case "tags":
+			return {
+				details: "Browsing Community forums...",
+				state: "Filter tags",
+				smallImageKey: Assets.Searching,
+			};
+		case "c": {
+			// category, topic and post forum level
+			return subtitle
+				? {
+						details: `Forum: ${pageDetail.replaceAll("-", " ")}`,
+						state: `Topic: ${title.replaceAll("-", " ")}`,
+						smallImageKey: Assets.Searching,
+				  }
+				: {
+						details: "Browsing forum category...",
+						state: pageDetail.replaceAll("-", " "),
+						smallImageKey: Assets.Searching,
+				  };
+		}
+		case "t": {
+			return {
+				details: `Forum: ${
+					document.querySelector(
+						"a:nth-child(1) > span.badge-category.clear-badge > span"
+					).textContent
+				} > ${
+					document.querySelector(
+						"a:nth-child(2) > span.badge-category.clear-badge > span"
+					).textContent
+				}`,
+				state: `Post: ${document.querySelector(".fancy-title").textContent}`,
+				smallImageKey: Assets.Reviewing,
+			};
+		}
+		case "about": // general memrise domain
+			return { details: "Viewing About us page..." };
+		case "team":
+			return { details: `Viewing Memrise ${page}...` };
+		case "philosophy":
+			return { details: `Reading Memrise ${page}...` };
+		case "jobs":
+			return { details: `Searching Memrise ${page}...` };
+		case "blog":
+			return {
+				details: `Reading Memrise ${page}s...`,
+				state: document.querySelector("#hs_cos_wrapper_name").textContent, //blog title
+				smallImageKey: Assets.Reviewing,
+			};
+		case "contact":
+			return { details: `Viewing Memrise ${page} page` };
+		case "terms":
+			return { details: `Reading Memrise ${page}` };
+		case "privacy":
+			return { details: `Reading Memrise ${page} policy` };
+		case "cookies":
+			return { details: `Reading Memrise ${page} policy` };
+		default:
+			return { details: "Browsing...", smallImageKey: Assets.Searching };
+	}
+}
+function getHomeDetail(pageDetail: string) {
+	switch (pageDetail) {
+		case "learning-statistics":
+			return {
+				details: "Viewing their learning stats...",
+				state: `${document.querySelector(".rank").textContent} 
+				[Best Streak ${
+					document.querySelector("#attendance-grid-label2 .large").textContent
+				} | 
+				Current ${
+					document.querySelector("#attendance-grid-label3 .large").textContent
+				} | 
+				Total ${
+					document.querySelector("#attendance-grid-label1 .large").textContent
+				} days the last 365 days]`,
```

Comment:
You sure this sentence is correct?

Reply:
Yes, should be the total numbers of days you have learned the past 365 days. So in short: "Total 36 days the last 365 days". Unless there is a better sentence.

Reply:
Maybe add `in` so it is `... days in the last 365 days`

Reply:
Good one, I'll change it soon